### PR TITLE
feat: the aspects code work, but i will make it clean later

### PR DESCRIPTION
**Pull request by aldoprogrammer**
## Related Issue(s)

[//]: # ([Cite any related issue&#40;s&#41; or feature request&#40;s&#41; from the repository's issue tracker.])
Fixes #
For #

## Proposed Changes

[//]: # ([List down the specific changes you have made in this pull request.])

- [x]
- [x]

## Checklist

- [x] I have followed the repository's contribution guidelines.
- [x] I have added necessary documentation or updated existing documentation.
- [x] My code follows the established code style and formatting of the project.
- [x] I have tested my changes and they function as expected.
- [ ] I have added suitable unit tests (if applicable).

## To-Do

- [ ]

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
     distDir: 'build',
-    output: 'export',
+    // output: 'export',
```
Comment (Nasfame) [inline next.config.js:4]: why disable it? Where r using nextjs exclusively for frontend only

**Review by Nasfame — CHANGES_REQUESTED** [review]

**Comment by Nasfame** [comment]
@aldoprogrammer fix the vercel logs. I hope u r able to access the logs!!

File: next.config.js (hunk 1 above)
Comment (aldoprogrammer) [reply next.config.js]: bcs if i dont disable it, the code itself won't work on my machine. i tried to debug it, and found on the internet to remvoe that line, that's why i commented it. and if i dont commeneted it, it wont work on my machine.

**Comment by aldoprogrammer** [comment]
> @aldoprogrammer fix the vercel logs. I hope u r able to access the logs!!

I'll fix everything later on

File: next.config.js (hunk 1 above)
Comment (aldoprogrammer) [reply next.config.js]: I fixed it now, but adding the images: { unoptimized: true }, and not comenting the output anymore, tht is the only two choices which the nextjs itself gives an error to fixed it when i tried to run the code

Comment (Nasfame) [reply next.config.js]: ok since we are on vercel we can support SSR. So no worries of static builds. But fix the rest of issues in vercel deployment

**Comment by Nasfame** [comment]
@aldoprogrammer 

I would advice u to follow a  next13 tutorial. 
- Understand routing in next13
- Build apis in nextJS



We are turning down this pr cuz u have invalid code in a NextJS Project. For instance,  usage of React router is not required.

**Comment by aldoprogrammer** [comment]
@Nasfame Sure, I am so sorry for this mess cause I haven't learned the next js to be honest, only react js before this. But,now I already watch some Next js tutorial, and hopefuly will not making any mess anymore in this project. 

Anyway, the todos in nextjs13 app you asked me for already done, i already give the link in the attlassian.

**Pull request closed without merging**